examples/screenshot.rs

```diff
@@ -6,28 +6,8 @@ use libremarkable::framebuffer::*;
 use libremarkable::image::RgbImage;
 use std::fs::OpenOptions;
 
-fn to_srgb([r, g, b]: [u8; 3]) -> [u8; 3] {
-    // This is a mapping from Linear RGB values to sRGB values, required for output to normal images
-    // The calculation performed was: ((i as f32 / 255.0).powf(1.0 / 2.2) * 255.0) as u8
-    const GAMMA: [u8; 256] = [
-        0, 20, 28, 33, 38, 42, 46, 49, 52, 55, 58, 61, 63, 65, 68, 70, 72, 74, 76, 78, 80, 81, 83,
-        85, 87, 88, 90, 91, 93, 94, 96, 97, 99, 100, 102, 103, 104, 106, 107, 108, 109, 111, 112,
-        113, 114, 115, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 128, 129, 130, 131, 132,
-        133, 134, 135, 136, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 147, 148,
-        149, 150, 151, 152, 153, 153, 154, 155, 156, 157, 158, 158, 159, 160, 161, 162, 162, 163,
-        164, 165, 165, 166, 167, 168, 168, 169, 170, 171, 171, 172, 173, 174, 174, 175, 176, 176,
-        177, 178, 178, 179, 180, 181, 181, 182, 183, 183, 184, 185, 185, 186, 187, 187, 188, 189,
-        189, 190, 190, 191, 192, 192, 193, 194, 194, 195, 196, 196, 197, 197, 198, 199, 199, 200,
-        200, 201, 202, 202, 203, 203, 204, 205, 205, 206, 206, 207, 208, 208, 209, 209, 210, 210,
-        211, 212, 212, 213, 213, 214, 214, 215, 216, 216, 217, 217, 218, 218, 219, 219, 220, 220,
-        221, 222, 222, 223, 223, 224, 224, 225, 225, 226, 226, 227, 227, 228, 228, 229, 229, 230,
-        230, 231, 231, 232, 232, 233, 233, 234, 234, 235, 235, 236, 236, 237, 237, 238, 238, 239,
-        239, 240, 240, 241, 241, 242, 242, 243, 243, 244, 244, 245, 245, 246, 246, 247, 247, 248,
-        248, 249, 249, 249, 250, 250, 251, 251, 252, 252, 253, 253, 254, 254, 255,
-    ];
-
-    [GAMMA[r as usize], GAMMA[g as usize], GAMMA[b as usize]]
-}
+mod rgb565_to_srgb;
+use rgb565_to_srgb::RGB565_TO_RGB888;
```

Comment:
At this point wouldn't it make sense to actually publish this where it can be reused by other things than this example?

Reply:
Yes, it would, which is actually why I've been working on a crate for it for the past few hours. It should be able to replace libremarkable's home-grown rgb565 functions and also provide some runtime efficiency via LUTs generated at compile-time.

It includes rgb565_le, as well as some common transformations with optional embedded look-up tables.

Reply:
Cool.
Dang I'm curious what the compile times are on your https://docs.rs/rgb565/latest/rgb565/

Reply:
A cold debug build takes 1.32s. A cold release build takes 2.24s. Without the 32MB LUTs enabled of course. With those enabled, debug build takes 21s and release build takes 17.06s. But they're disabled by default.

Reply:
A cold debug build takes 1.32s. A cold release build takes 2.24s. Without the 32MB LUTs enabled of course. With those enabled, debug build takes 21s and release build takes 17.06s. But they're disabled by default.